README.md

```diff
@@ -2,11 +2,13 @@
 
 This project provides a Redis storage mechanism for [Bot Framework-JS SDK V4.](https://github.com/Microsoft/botbuilder-js).
 
-It allows you to store bot state in Redis, so that you can scale out your bot, and be more resilient to bot server failures.
+It allows you to store bot state in Redis, so that you can scale out your bot,
```

Comment:
@mokeddes Your pre-commit hook setup is quite impressive, but it forced me to reformat README.md because markdownlint doesn't allow lines longer than 80 characters.
Commitlint forced me to use `ttlinseconds` in commit message even though `ttlInSeconds` would be more accurate.

Reply:
@Naktibalda  i this we dont need setAsync method, we need only setExpAsync because they make the same job if ttl=0

Reply:
I haven't found any documentation about that.

Reply:
I removed setAsync